test/zepto.html

```diff
@@ -1653,12 +1653,18 @@
 
       testCSSOnNonExistingElement: function (t) {
         var errorWasRaised = false
+        var el = $('.some-non-exist-elm')
         try {
-          var color = $('.some-non-exist-elm').css('color')
+          var color = el.css('color')
+          var ret = el.css({
```

Comment:
This is a good test, but can you add a separate test case for it? Maybe call it something like `testCSSOnNonExistingElementReturnsSelf`. You also don't need `try…catch` blocks in that test case. Then you can leave this test case unchanged.